04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=0.7\textwidth]{04_environment/images/Pareto Optimal Strategy.PNG}
+    \caption{Pareto Optimal Strategy}
+    \label{fig:Pareto Optimal Strategy}
+\end{figure}
+
+\subsubsection{Social Welfare Maximisation}
+
+It is possible for these islands to find the social welfare maximum (see Definition~\ref{def:Maximising Social Welfare}) in this foraging function with enough time. The islands could identify the exact amount of resources needed, for both fishing and deer hunting, to achieve the best return according to the maximum number of animals. Thus, by cooperating who goes where and spends how much, they are able to maximize their returns without spending more resources than necessary.
+
+\subsubsection{Population Density}
+
+Introducing a model that controls the population density of deer allows us to increase the complexity of the foraging function. In short, the deer hunting capacity decreases with decreasing population and increases with an increasing population. This then influences the maximum deer per hunt parameter ($n$), which in turn results in a more complex return calculation function (\texttt{\textbf{DtotalReturn}}), which would make it harder for agents to figure out forage tier boundaries to optimise their strategies.
+
+The population change can be dynamic depending on various environmental factors, fetched from the rest of the system. For example, a disaster could supposedly cause the population density of deer to halve, making it harder to forage and therefore, causing lower returns. In this current implementation, a single species population model and more specifically logistic modelling for the deer and fish population was implemented. 
+The model is described in Equation~\eqref{eq:Population Density}.
+
+\begin{equation}
+\frac{\mathrm{d} P}{\mathrm{~d} t}=k(N-P)
+\label{eq:Population Density}
+\end{equation}
+
+\begin{itemize}
+    \item $P$ is the total population. 
+    \item $N$ is the maximum deer population (carrying capacity\footnote{The maximum population size of a biological species that can be sustained in a specific environment.}).
+    \item $k$ is the growth coefficient.
+    \item $t$ is time.
+\end{itemize}
+
+In Figure~\ref{fig:Deer population over time}, the simulation of the implemented logistic model can be observed, with a growth coefficient of $0.2$ and $N$ of $8$. It can be seen that there is an unpredictable pattern to population changes, which would be interesting to see affecting the tier system and in extend, the deer and fish foraging strategies of the agents.
```

Comment:
no need to put numbers in $8$ - it doesn't change how they appear and can cause bugs depending how a doc is set up